### PR TITLE
Add the integrate wallet guide

### LINKED ISSUE #417: Explain how to integrate the wallet service in a CI/CD
This can be useful for dApps devs that want to work with the wallet.

This is a guide how to set up  from scratch and use Vega Wallet for Vega dApps. You can find current working CI/CD implementation [here](https://github.com/vegaprotocol/frontend-monorepo/blob/develop/.github/actions/setup-vegawallet/action.yml). Some steps may be skipped - api-token related steps are needed only for E2E and CI/CD integration. Also, if you already have Vega wallet prepared and configured, you will probably need only the last step to run the wallet.

1. Create two files: `passphrase` and `recovery` - `passphrase` will be used as a wallet and api-token password (you may want to use two different ones), and `recovery` should contain recovery phrase of the wallet you want to use (you may skip it if you don’t have one yet, and want to create a new one.
2. Download and install Vega wallet. There are two ways to do that:
    1. Download the latest release of [Vega](https://github.com/vegaprotocol/vega/releases) and unzip it to some shared location. To test if it worked you may run `vega version` command.
    2. Clone [Vega repo](https://github.com/vegaprotocol/vega). Make sure you have installed `go` (`1.19` version required at the time of writing this document). Setup `GOBIN` env and add it to `PATH` (e.g. `export GOBIN="$HOME/go/bin; export PATH=$GOBIN/:$PATH`. Run `go install ./…` to install binaries. Verify by running `vega version` command.
3. Initialise a wallet - `vega wallet init -f --home <WALLET_PATH>` - this one creates new, fresh wallet files. 
`--home <WALLET_PATH>` sets wallet location. You can skip it if you want to use default location, you should skip it in all other steps as well then. 
`-f` flag overwrites existing wallet configuration at the location. 
4. Create new or import existing wallet (you may create/import as many wallets as you want).
    1. New wallet: `vega wallet create -w <WALLET_NAME> -p passphrase --home  <WALLET_PATH>`
    2. Import the wallet using recovery phrase:  `vega wallet import -w <WALLET_NAME> --recovery-phrase-file recovery -p passphrase --home <WALLET_PATH>`.
5. You may create additional public keys if you need - `vega wallet key generate -w <WALLET_NAME> -p passphrase --home <WALLET_PATH>`
6. Import network config. Again, two ways to do that:
    1. Using URL: `vega wallet network import --from-url <CONFIG_URL> -f --home <WALLET_NAME>`.  `-f` flag overwrites the config if existed. Example: to import `fairground` network you should use: `vega wallet network import --from-url https://github.com/vegaprotocol/networks-internal/blob/main/fairground/vegawallet-fairground.toml`.
    2. Using file: If you want to import network for local vegacapsule network use this one: `vega wallet network import --force --from-file <CONFIG_FILE> --home <WALLET_PATH>`
7. If you need to obtain long live token for testing purposes you should do the following:
    1. Firstly initialise the api-token by command: `vega wallet api-token init --passphrase-file passphrase --home <WALLET_PATH>`
    2. Then generate the api-token for a single wallet: `vega wallet api-token generate --wallet-name <WALLET_NAME> --tokens-passphrase-file passphrase  --wallet-passphrase-file passphrase --home <WALLET_PATH>`
    3. Copy generated api-token and use it in your dAPP, tests or CI. It is the token that is normally obtained by `client.connect_wallet` request.
8. And now to start the service you need to use:
    1. For general purpose it is enough to use: `vega wallet service run -n <NETWORK_NAME> --home <WALLET_PATH>`.
    2.  In tests or CI more complex command should be used: `vega wallet service run -n <NETWORK_NAME> --load-tokens --tokens-passphrase-file passphrase --no-version-check --automatic-consent --home <WALLET_PATH>`.
     `--load-tokens` allows to use the token that was set up in previous steps. 
     `--no-version-check` allows to skip verification of network - if it’s alive and if wallet version matches network version. 
     `--automatic-consent` allows wallet to automatically accepts any requests and transactions that’s coming from connected apps.

And that’s it. The flow looks quite long, but you don’t really need to go through all the steps every time. If you don’t use it for CI or Testing purposes, you can easily skip few of the steps that relate to api-token. Also after you set up everything, you will probably only need to run `service run` command and update the binaries when the new version is released. Vega wallet HELP is pretty well documented. If you’re stuck with some command, just use e.g. `vega  wallet -h` or `vega wallet api-token generate -h` and you will receive all the information you need about the command. I hope you will find this guide helpful.

### PR DESCRIPTION
**Pull request by candida-d**
Closes #417 

Open question: 
Is the wallet api V2 section the best place for this guide?

### CODE REVIEW

File: docs/api/vega-wallet/v2-api/integrate-wallet-service.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+You can find the current working CI/CD implementation in the [front-end GitHub repo ↗](https://github.com/vegaprotocol/frontend-monorepo/blob/develop/.github/actions/setup-vegawallet/action.yml).
+
+Depending on your end-goal, you may be able to skip some steps. API token-related steps are only needed for end-to-end and CI/CD integration. Also, if you already have Vega Wallet prepared and configured, you may only need the last step to run the wallet.
+
+1. **Create two files for your `passphrase` and `recovery` phrase** - `passphrase` will be used as a wallet and API token password (you may want to use two different ones), and `recovery` will contain recovery phrase of the wallet you want to use (you can skip it if you don’t have a wallet yet, and want to create a new one).
+
+2. **Download and install the Vega Wallet software.** 
+    There are two ways you can do that:
+    a. Download the latest release of the [Vega software ↗](https://github.com/vegaprotocol/vega/releases) and unzip it to a shared location. To test if it worked, run the `vega version` command.
```
Comment (ValentinTrinque) [inline docs/api/vega-wallet/v2-api/integrate-wallet-service.md:17]: The command is now 

Vega software check

File: docs/api/vega-wallet/v2-api/integrate-wallet-service.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    b. Clone the [Vega repo ↗](https://github.com/vegaprotocol/vega). Make sure you have installed `go`. Read the guide for [getting started with Go for Vega ↗](https://github.com/vegaprotocol/vega/blob/develop/GETTING_STARTED.md#installing-golang) if you need help. Set up the `GOBIN` environment and add it to `PATH` (e.g. `export GOBIN="$HOME/go/bin; export PATH=$GOBIN/:$PATH`. Run `go install ./…` to install the binaries. Verify by running the command `vega version`.
```
Comment (ValentinTrinque) [inline docs/api/vega-wallet/v2-api/integrate-wallet-service.md:18]: Go install ./cmd/vegawallet ./cmd/vega

File: docs/api/vega-wallet/v2-api/integrate-wallet-service.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    vega wallet service run -n <NETWORK_NAME> --load-tokens --tokens-passphrase-file passphrase --no-version-check --automatic-consent --home <WALLET_PATH>
+    ````
+
+    `--load-tokens` allows you to use the API token that you set up in the previous step.
+    `--no-version-check` allows you to skip network compatibility verification.
+    `--automatic-consent` allows the wallet to automatically approve any transactions that come from connected dApps.
+
+Once you set up a wallet for the first time, you will probably only need to run the `service run` command and update the binaries when a new version is released. 
+
+**If you need help using the wallet software**, use the Vega Wallet `--help` flag. If you’re stuck on a command, use `vega wallet -h` for general guidance, or, for example, `vega wallet api-token generate -h` to read details about a specific command.
```
Comment (ValentinTrinque) [inline docs/api/vega-wallet/v2-api/integrate-wallet-service.md:101]: ❤️

**Review by ValentinTrinque — CHANGES_REQUESTED** [review]

File: docs/api/vega-wallet/v2-api/integrate-wallet-service.md (hunk 2 above)
Comment (radoslaw-szpiech) [reply docs/api/vega-wallet/v2-api/integrate-wallet-service.md:18]: @ValentinTrinque does not `go install ./...` do the same?

File: docs/api/vega-wallet/v2-api/integrate-wallet-service.md (hunk 1 above)
Comment (radoslaw-szpiech) [reply docs/api/vega-wallet/v2-api/integrate-wallet-service.md:17]: @ValentinTrinque did something change? As for `v0.67.3` there are two commands available
- `vega version` - it returns version of vega binaries (SHOULD be the same as vegawallet binaries version)
- `vega wallet software version` - it returns vegawallet binaries version

There is no such command as `vega software check`.

Comment (ValentinTrinque) [reply docs/api/vega-wallet/v2-api/integrate-wallet-service.md:17]: Sorry I meant:

vega **wallet** software check

File: docs/api/vega-wallet/v2-api/integrate-wallet-service.md (hunk 2 above)
Comment (ValentinTrinque) [reply docs/api/vega-wallet/v2-api/integrate-wallet-service.md:18]: I shall verify but I am not sure.

Comment (ValentinTrinque) [reply docs/api/vega-wallet/v2-api/integrate-wallet-service.md:18]: YOu are right, it's properly resolving the project mains.

**Review by ValentinTrinque — APPROVED** [review]

**Pull request merged**